models/DataObject/ClassDefinition/Data/Extension/Relation.php

```diff
@@ -39,7 +39,7 @@ protected function getPhpDocClassString($asArray = false)
                 $class[] = '\Pimcore\Model\Document' . $strArray;
             } elseif (is_array($documentTypes)) {
                 foreach ($documentTypes as $item) {
-                    $class[] = sprintf('\Pimcore\Model\Document\%s', ucfirst($item['documentTypes']) . $strArray);
+                    $class[] = $this->getMappedClassName('\Pimcore\Model\Document\\' . ucfirst($item['documentTypes']), $strArray);
```

Comment:
```suggestion
                    $class[] = $this->getMappedClassName('\Pimcore\Model\Document\\' . ucfirst($item['documentTypes'])) . $strArray;
```

Reply:
This will probably reduce conflicts with the `v11` branch, because there the `$strArray` gets added later. See: #13934